### ISSUE TITLE
core lldp tlv population test issues

### ISSUE THREAD

**Issue by Hobbydos** [openconfig/featureprofiles#303]
The want/got values expect to be validated for the same time, if the validation happens 1 second after, it fails

```
core_lldp_tlv_population_test.go:93: /interfaces/interface[name=int1]/neighbors/neighbor[id=01:01:01:01:01:01]/state/age: got &441, want &440
core_lldp_tlv_population_test.go:93: /interfaces/interface[name=int1]/neighbors/neighbor[id=01:01:01:01:01:01]/state/last-update: got &12, want &11
```

There also seems to be issues fetching certain values which are present when querying the paths in gNMI:

```
core_lldp_tlv_population_test.go:126: /state/port-id-type: got <nil>, want INTERFACE_NAME
```
call  via gNMI
```
gnmic -a 10.254.103.19 -u admin -p admin --skip-verify get --path "/lldp/interfaces/interface[name=int1]/neighbors/neighbor" -e json_ietf
[...output trimmed...]
                  "last-update": "7",
                  "port-description": "",
                  "port-id": "int1",
                  "port-id-type": "INTERFACE_NAME",
[...output trimmed...]
```

the same is true for the lldp interface state:

```
core_lldp_tlv_population_test.go:93: /interfaces/interface[name=int1]/state/enabled: got <nil>, want &true
```

```
gnmic -a 10.254.103.19 -u admin -p admin --skip-verify get --path "/lldp/interfaces/interface[name=int1]/state/enabled" -e json_ietf
[...output trimmed...]
        "Path": "openconfig-lldp:lldp/interfaces/interface[name=int1]/state/enabled",
        "values": {
          "openconfig-lldp:lldp/interfaces/interface/state/enabled": true
```

**Comment by Hobbydos** [comment]
@hariramnat  Have you had a chance to look into this case?
I also noticed that all my outputs were done with `get`, here's an updated output using `sub once` regarding `INTERFACE_NAME` and `state/enabled`
```
gnmic -a 10.254.103.19 -u username -p password --skip-verify sub --path "/lldp/interfaces/interface[name=int1]/neighbors/neighbor" --mode once -e json_ietf    
[...output trimmed...]
          "last-update": "11",
          "port-description": "",
          "port-id": "int1",
          "port-id-type": "INTERFACE_NAME",
[...output trimmed...]
```


```
gnmic -a 10.254.103.19 -u username -p password --skip-verify sub --path "/lldp/interfaces/interface[name=int1]/state" --mode once -e json_ietf
[...output trimmed...]
        "openconfig-lldp:lldp/interfaces/interface/state": {
          "enabled": true,
          "name": "int1"
[...output trimmed...]
```

**Comment by Hobbydos** [comment]
Closed due to merge on #415
